source/common/http/conn_manager_config.h

```diff
@@ -35,6 +35,7 @@ namespace Http {
   COUNTER(downstream_cx_http3_total)                                                               \
   COUNTER(downstream_cx_idle_timeout)                                                              \
   COUNTER(downstream_cx_max_duration_reached)                                                      \
+  COUNTER(downstream_cx_max_requests)                                                              \
```

Comment:
rename to `downstream_cx_max_requests_reached`

Reply:
Done